about.html

```diff
@@ -12,53 +12,122 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
```

Comment:
You should try to use semantic HTML the header/nav is a great please to use the `<header> and <nav>` elements.